web/toolbar.js

```diff
@@ -340,7 +344,7 @@ class Toolbar {
     ctx.font = `${style.fontSize} ${style.fontFamily}`;
 
     let maxWidth = 0;
-    for (const predefinedValue of await predefinedValuesPromise) {
+    for (const { predefinedValue } of await predefinedValuesPromise) {
```

Comment:
This doesn't work in the viewer, when testing with the `de` locale. However the following seems correct:
```suggestion
    for (const predefinedValue of await predefinedValuesPromise) {
```